tuf/client/client.go

```diff
@@ -111,10 +110,6 @@ func (c Client) checkRoot() error {
 		return fmt.Errorf("Cached root sha256 did not match snapshot root sha256")
```

Comment:
Agreed that we should fix this -- I think `memorystore.go` is fine so it'll only be a change to `filestore.go`